client/src/components/game/star/StarDetail.vue

```diff
@@ -447,6 +467,24 @@ export default {
       }
 
       return GameHelper.getStarById(this.$store.state.game, this.star.wormHoleToStarId)
+    },
+    naturalResources: function() {
+      if  (!splitResources) {
+        return Math.floor(this.star.naturalResources.economy);
+      }
+      let economy = Math.floor(this.star.naturalResources.economy);
+      let industry = Math.floor(this.star.naturalResources.industry);
+      let science = Math.floor(this.star.naturalResources.science);
```

Comment:
This stuff should be floored by the game galaxy API response instead of on the client.

Reply:
Resolved in the client

Reply:
resolved in server